site/docs/v1/tech/reference/cookies.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: doc
+title: Hosted Login Pages Cookies
+description: A list of cookies set by the FusionAuth Hosted Login pages
+navcategory: admin
+---
+
+== Overview
+
+Cookies are a critical part of web applications, and FusionAuth sets cookies. When you call certain APIs, such as the Login API, cookies may be set. These cookies are specified in the link:/docs/v1/tech/apis[API documentation].
+
+When you use the link:/docs/v1/tech/core-concepts/integration-points#hosted-login-pages[hosted login pages], FusionAuth sets cookies to enable functionality.
+
+== Domains
+
+The domain of all cookies is limited to the domain on which the FusionAuth instance is running. You can control the domain FusionAuth uses by link:/docs/v1/tech/admin-guide/proxy-setup[setting up a proxy].
+
+In other words, if FusionAuth lives at `auth.piedpiper.com`, it will only set cookies for that domain. It will never set cookies for `.piedpiper.com`.
+
+== Cookie List
+
+[WARNING.warning]
+====
+Cookies set by the hosted login pages are displayed here for informational purposes only.
+
+Duration, name, and purpose are subject to change.
+====
+
+When a cookie has a value of `Yes` for the `Prefix` column, name is used as a prefix for one or more cookies.
+
+//Pulled from Cookies.java
```

Comment:
We should probably categorize these. Many of these cookies are very transient and only used for a request and then deleted, an in theory you may not see them, and even if you delete them it wouldn't hurt as long as that specific request has completed. 

Examples of these are `fusionauth-app.pkce.verifier`, and `fusionauth.pkce-verifier`. 

Should we only document cookies that are persisted for any length of time? And from a GDPR point of view, as long as we are doing this - should we indicate if these are "preference", "required" or "tracking" etc?

Reply:
Sure, happy to mark cookies that are request based as 'request' instead of 'session'. We could omit them, as you suggest, and just throw some language in saying 'There may be other transient cookies set by FusionAuth' or something similar, but then the question is 'well, what are those transient cookies doing'.

As far as the GDPR stuff goes, I'd rather just let the end user determine whether things are required based on our broad descriptions, at least for v1. If the client requesting this info comes back and says 'please categorize these' then I'd be okay with further categorization.

We should also specify that by using themes the client may add their own cookies (using javascript for analytics) but that is outside the scope of this doc.